lexicon/modification.py

```diff
@@ -0,0 +1,156 @@
+__all__ = ["StaticLexiconJob", "MergeLexiconJob"]
+
+from collections import OrderedDict
+
+from sisyphus import Job, Task
+
+from i6_core.lib import lexicon
+from i6_core.util import write_xml
+
+
+class StaticLexiconJob(Job):
```

Comment:
I think a better name would be `WriteLexiconJob` because it effectively only writes out a given lexicon.

Reply:
agree